OP2Helper.h

```diff
@@ -52,9 +52,25 @@ bool operator!= (const LOCATION& loc1, const LOCATION &loc2);
 // Starting Resources
 void InitPlayerResources(int playerNum, const ResourceSet& resourceSet = CES1ResourceSet);
 
-// Constructing Tubes/Walls
+
+// Create wall or tubes in a straight line or L pattern
+// L patterns draw counter - clockwise between the two points
+void CreateTubeLine(LOCATION loc1, LOCATION loc2);
+void CreateWallLine(LOCATION loc1, LOCATION loc2);
+void CreateLavaWallLine(LOCATION loc1, LOCATION loc2);
+void CreateMicrobeWallLine(LOCATION loc1, LOCATION loc2);
+
+// Record line of tubes or walls to a BuildingGroup in a straight line or L pattern
+// L patterns draw counter - clockwise between the two points
+void RecordTubeLine(BuildingGroup& buildingGroup, LOCATION loc1, LOCATION loc2);
+void RecordWallLine(BuildingGroup& buildingGroup, const LOCATION& loc1, const LOCATION& loc2);
+void RecordLavaWallLine(BuildingGroup& buildingGroup, const LOCATION& loc1, const LOCATION& loc2);
+void RecordMicrobeWallLine(BuildingGroup& buildingGroup, const LOCATION& loc1, const LOCATION& loc2);
+
+[[deprecated("CreateTubeOrWallLine is deprecated. Use CreateWallLine or CreateTubeLine instead.")]]
 void CreateTubeOrWallLine(int x1, int y1, int x2, int y2, map_id type);

```

Comment:
This function is instantiated so that it can be wholesale deleted in a future version without affecting any other function/code within OP2Helper.

Reply:
Really minor point: For technical writing we should probably try to list things in the same order whenever possible, hence "TubeOrWall" -> "Tube", "Wall". Might also be good to indicate there are multiple wall types. Perhaps:
```cpp
[[deprecated("CreateTubeOrWallLine is deprecated. Use CreateTubeLine or Create[|Lava|Microbe]WallLine instead.")]]
```
Alternatively maybe `Create<X>WallLine` might be another syntax. I'm not overly against leaving it simply as `CreateWallLine` though.